include/llvm/Config/config.h

```diff
@@ -51,22 +51,30 @@
 #define LLVM_ENABLE_DIA_SDK 0
 
 /* Define to 1 if you have the <dlfcn.h> header file. */
+#if __has_include(<dlfcn.h>)
 #define HAVE_DLFCN_H 1
 
 /* Define if dlopen() is available on this platform. */
 #define HAVE_DLOPEN 1
 
 /* Define if dladdr() is available on this platform. */
 #define HAVE_DLADDR 1
+#endif
 
 /* Define to 1 if you have the <errno.h> header file. */
+#if __has_include(<errno.h>)
 #define HAVE_ERRNO_H 1
+#endif
 
 /* Define to 1 if you have the <fcntl.h> header file. */
+#if __has_include(<fnctl.h>)
```

Comment:
The "n" and "c" are swapped here, that's what broke the build.

Reply:
Ah, fixed.